internal/providers/pluginfw/resources/qualitymonitor/resource_quality_monitor_test.go

```diff
@@ -0,0 +1,153 @@
+package qualitymonitor_test
```

Comment:
I think this should be qualitymonitor only? 

Also, might be good to move volumes data source test to under volume package as well.

Reply:
This is an interesting point. There's some circular dependency issue with it just being `qualitymonitor` because the `init.go` file imports some provider factory, and provider factory indirectly imports from `qualitymonitor`

Reply:
So I think the easiest workaround is just to give the test file a different package name